test/integration/disruption/disruption_test.go

```diff
@@ -507,6 +509,21 @@ func waitToObservePods(t *testing.T, podInformer cache.SharedIndexInformer, podN
 	}
 }
 
+// createPDBUsingRemovedAPI creates a PDB directly using etcd.  This is must *ONLY* be used for checks of compatibility
+// with removed data. Do not use this just because you don't want to update your test to use v1.  Only use this
+// when it actually matters.
+func createPDBUsingRemovedAPI(ctx context.Context, etcdClient *clientv3.Client, etcdStoragePrefix, nsName string, betaPDB *v1beta1.PodDisruptionBudget) error {
+	betaPDB.APIVersion = v1beta1.SchemeGroupVersion.Group + "/" + v1beta1.SchemeGroupVersion.Version
+	betaPDB.Kind = "PodDisruptionBudget"
+	betaPDB.Namespace = nsName
+	betaPDB.Generation = 1
+	ctx = genericapirequest.WithNamespace(ctx, nsName)
+	key := path.Join("/", etcdStoragePrefix, "poddisruptionbudgets", nsName, betaPDB.Name)
+	val, _ := json.Marshal(betaPDB)
+	_, err := etcdClient.Put(ctx, key, string(val))
```

Comment:
to match actual API server behavior, this should marshal to protobuf to persist in etcd, not json.

something like this could work:

```go
protoSerializer := protobuf.NewSerializer(legacyscheme.Scheme, legacyscheme.Scheme)
buffer := bytes.NewBuffer()
if err := protoSerializer.Encode(betaPDB, buffer); err != nil {
  return err
}
... etcdClient.Put(ctx, key, string(buffer.String()))
```